packages/all/package.json

```diff
@@ -5,7 +5,7 @@
     "@govuk-frontend/back-link": "^0.0.19-alpha",
     "@govuk-frontend/breadcrumbs": "^0.0.19-alpha",
     "@govuk-frontend/button": "^0.0.19-alpha",
-    "@govuk-frontend/checkbox": "^0.0.19-alpha",
+    "@govuk-frontend/checkboxes": "^0.0.20-alpha",
```

Comment:
Is this bump intentional when renaming?

Reply:
As far as NPM is concerned, we've published it a new package, not renamed one.

@nickcolley was looking into this and added a note to dev time to discuss how we should handle deleted NPM packages (which govuk-frontend/checkbox will be)